examples/persistence_service/persistent_storage/c++11/hello_world_subscriber.cxx

```diff
@@ -44,23 +44,20 @@ class hello_worldReaderListener : public NoOpDataReaderListener<hello_world> {
     }
 };
 
-void subscriber_main(int domain_id, int sample_count, bool drs)
+void subscriber_main(int domain_id, int sample_count, int is_persistent)
```

Comment:
```suggestion
void subscriber_main(int domain_id, int sample_count, bool is_persistent)
```

Reply:
I would prefer keeping it `int` since we use the function `atoi()` to parse the value.